build.gradle

```diff
@@ -39,7 +46,11 @@ dependencies {
     compile "org.eclipse.xtext:org.eclipse.xtext:$versions.xtext"
     compile "io.dropwizard:dropwizard-core:$versions.dropwizard"
     compile "org.eclipse.jgit:org.eclipse.jgit:$versions.jgit"
+	compile "org.testeditor.web:org.testeditor.web.dropwizard:$versions.testEditorDropwizard"
+	// compile "org.testeditor.web:org.testeditor.web.dropwizard.xtext:$versions.testEditorDropwizard"
 
+    testCompile 'org.testeditor:org.testeditor.tsl.dsl.web:1.7.0'
```

Comment:
(this line and the next) go to 1.8.0? And maybe introduce variable "testEditorLanguage" for the version